src/components/messenger/list/index.tsx

```diff
@@ -241,12 +239,14 @@ export class Container extends React.Component<Properties, State> {
 
         {this.props.stage === SagaStage.None && (
           <div {...cnMessageList('profile-bar')}>
-            {this.props.includeRewardsAvatar && (
+            {this.props.isMessengerFullScreen && (
```

Comment:
I think this would work better as `includesAvatar` or `includesUserSettings` or whatever it is. The reasoning for this is that if we write our components with as little knowledge of what the overall app is structured like then they're simpler to move around and reuse, etc. So, this component probably shouldn't even know whether we're in full screen or not. It should only know how the app wants it to be rendered. From that perspective the higher level component is saying, "in this location with these conditions `includeUserSettings` when you're rendered".